scripts/json-schema-dereference.bash

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit
+set -o nounset
+set -o pipefail
+IFS=$'\n\t'
+
+image_name="$(basename $0):latest"
+
+# derefernce json-schemas for easy comparison
+# SEE https://github.com/davidkelley/json-dereference-cli
+docker build --tag "$image_name" - <<EOF
+FROM node:12.18.2
+RUN npm install -g json-dereference-cli
```

Comment:
maybe having the version pinned of this `json-deference-cli` would help in the future.